lib/modules/manager/circleci/readme.md

```diff
@@ -1,3 +1,29 @@
 The `circleci` manager extracts both `docker` as well as `orb` datasources from CircleCI config files.
 
 If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+
+## Private orbs
```

Comment:
Our test doesn't like my `h2`:

```
 FAIL lib/modules/manager/metadata.spec.ts (331 MB heap size)
  ● modules/manager/metadata › circleci has readme with no h1 or h2
    expected true to be false or Boolean(false)
      31 |     expect(
      32 |       res.some((line) => line.startsWith('# ') || line.startsWith('## '))
    > 33 |     ).toBeFalse();
         |       ^
      34 |   });
      35 | });
      36 |
      at lib/modules/manager/metadata.spec.ts:33:7
```

Should I use a `h3` or will that fail another test? 🙈

Reply:
yeah, try h3

Reply:
```suggestion
### Private orbs
```